docs/plugins/main.py

```diff
@@ -111,31 +111,33 @@ def install_logfire(markdown: str, page: Page) -> str:
         # Split them and strip quotes for each one separately.
         extras = [arg.strip('\'"') for arg in arguments[1].strip('[]').split(',')] if len(arguments) > 1 else []
         package = 'logfire' if not extras else f"'logfire[{','.join(extras)}]'"
-        extras_arg = ' '.join(f'-E {extra}' for extra in extras)
+        # Hiding unused arg because the linter is yelling at me
+        #  extras_arg = ' '.join(f'-E {extra}' for extra in extras)
         instructions = f"""
-=== "PIP"
+=== "pip"
     ```bash
     pip install {package}
     ```
 
-=== "Rye"
+=== "uv"
     ```bash
-    rye add logfire {extras_arg}
+    uv add {package}
     ```
+"""
+        if not extras:
+            instructions += """
 
```

Comment:
This conditional needs to be only for conda, you need to move rye and poetry up.